### PR TITLE
feat: Dual login screen

### PR DESCRIPTION
**Pull request by ianjon3s**
## What does this PR do?

- Button style for login screen with icon
- Placeholder login button for Microsoft

### CODE REVIEW

**Review by DafyddLlyr — APPROVED** [review]
Perfect! We can merge this to main alongside @freemvmt's work once this is good to go 🙂

**Comment by ianjon3s** [comment]
[Allow Editors to login with their Microsoft ID](https://trello.com/c/owvV9UZh/2856-allow-editors-to-login-with-their-microsoft-id)

File: editor.planx.uk/src/pages/Login.tsx [hunk 1]
```diff
@@ -35,16 +43,21 @@ const Login: React.FC = () => {
...
         >
-          Login with Google
-        </Link>
+          <GoogleIcon />
+          Continue with Google
+        </LoginButton>
+        <LoginButton variant="contained" color="secondary" href={`...`}>
+          <MicrosoftIcon />
+          Continue with Microsoft
+        </LoginButton>
```
Comment (jessicamcinchak) [inline editor.planx.uk/src/pages/Login.tsx:60]: Shall we simply `disable` this button and adjust text to read something like `Continue with Microsoft (Coming soon)` and get this merged to `main` ahead of demo tomorrow & for future implementation work to rebase on top of? Might be easier than keeping an open PR around!

**Review by jessicamcinchak — APPROVED** [review]

File: editor.planx.uk/src/pages/Login.tsx (hunk 1 above)
Comment (ianjon3s) [reply editor.planx.uk/src/pages/Login.tsx:60]: Great shout, I've got a commit coming in for this.

Comment (DafyddLlyr) [reply editor.planx.uk/src/pages/Login.tsx:80]: Just came here to suggest this very thing! I'll then rebase #3221 for the demo tomorrow.

**Review by DafyddLlyr — APPROVED** [review]

**Pull request merged**